components/authz-service/storage/postgres/migration/migration.go

```diff
@@ -19,48 +15,7 @@ type Config struct {
 	Logger logger.Logger
 }
 
-type migrationLog struct {
-	logger.Logger
-}
-
-func (migrationLog) Verbose() bool {
-	return false
-}
-
 // Migrate executes all migrations we have
 func (c *Config) Migrate() error {
-	m, err := migrate.New(addScheme(c.Path), c.PGURL.String())
-	if err != nil {
-		return errors.Wrap(err, "init migrate")
-	}
-
-	m.Log = migrationLog{c.Logger} // nolint: govet
-
-	err = m.Up()
-	// Note: we've screwed up migration 25. If the state is dirty, and we're at
-	// version 25, reset to 24 and try again.
```

Comment:
Wondering about this custom patch... I infer that, once we have created migrations 26 and above, this redo code is no longer needed...?